CHANGELOG.md

```diff
@@ -3,9 +3,15 @@
 ## 2021-03-15
 * Added `st2chatops` support and service startup script. (#206)
 
+## 2021-03-14
```

Comment:
The date here is when the PR was merged.
In terms of chronology, it makes sense to include this PR after the `st2chatops` change which was included in `master` earlier.

Reply:
So what date do I change it to? I can't predict when the PR would be approved.

Reply:
Today's date is OK, the more important point is the Changelog order.